tests/README.md

```diff
@@ -1,26 +1,52 @@
 ## TDX Tests
 
-The tests should be run locally on TDX host.
+This folder contains TDX tests.
 
-It is assumed that these commands are running on a TDX host
-that has been setup properly.
+### Pre-requisites
 
-The guest image must be available at the location : `checkbox-provider-tdx/data/tdx-guest.qcow2`
-The guest image must enable ssh server with password-based authentication for `root` user.
-The root user password must be `123456`
+- The tests must be executed a TDX host that has been setup properly.
 
-Run tests with pytest
+- The guest image must be available at the project folder with the name `tdx-guest.qcow2`
+  You can follow the instructions in the project README to create one guest image.
+  You can specify a path to the guest image with `TDXTEST_IMAGE_FILE` environment variable.
+
+- The guest image must enable ssh server with password-based authentication for `root` user.
+  The root user password must be `123456`
+
+### Run tests with pytest
+
+- Run sanity tests to check the TDX host setup:
+
+```
+$ cd tests
+$ sudo ./run.sh pytest bin/test_host_*.py
+```
+
+- Run sanity tests to check the TDX guest boot:
 
 ```
-$ ./run.sh pytest
+$ cd tests
+$ sudo ./run.sh pytest bin/test_guest_*.py bin/test_boot_*.py
 ```
 
-Run tests with checkbox:
+- Run all tests except the performance ones:
 
 ```
-$ ./run.sh checkbox
+$ cd tests
+$ sudo ./run.sh pytest --ignore-glob *perf* bin/test_*.py
```

Comment:
This is not ignoring the performance tests for me. My exact commands:

```bash
~/wfrench/tdx/tests$ export TDXTEST_IMAGE_FILE=$HOME/wfrench/tdx/guest-tools/image/tdx-guest-ubuntu-24.04-intel.qcow2
~/wfrench/tdx/tests$ sudo -E ./run.sh pytest --ignore-glob *perf* bin/test_*.py
```

Reply:
I meant that the `bin/test_perf*` test were still being collected and run by pytest even with the `--ignore-glob` option. I just did some testing and I the following works for me:

```bash
./run.sh pytest --ignore-glob="bin/guest/" --ignore-glob="*perf*" bin/
```

For some reason using `bin/test_*.py` with `--ignore-glob` does not work. If you want to test there is a `--collect-only` you can pass to get pytest to basically do a dry run so you can see what is collected before it is run.

Reply:
Thanks !

Reply:
Oops - I forgot the `sudo` in my example command above and that made it into the README. Could you please update?